README.md

```diff
@@ -125,7 +125,7 @@ Then running `autobib get hochman-entropy` returns
 The record is identical to the record `zbl:1337.28015`, except that the citation key is the name of the alias.
 In order to distinguish from usual identifiers, an alias cannot contain the `:` colon symbol.
 
-Note that the characters `{}(),=\#%"` are not permitted in a BibTeX entry key.
+Note that the characters `{}(),=\#%"` and whitespaces are not permitted in a BibTeX entry key.
```

Comment:
Maybe 'whitespace characters' is better? "Whitespaces" feels weird to me. Then we could link to https://doc.rust-lang.org/reference/whitespace.html

i.e. [whitespace characters](https://doc.rust-lang.org/reference/whitespace.html)

Reply:
How about "whitespace" and link to the reference? I think this reads better than two "characters" in the same phrase.

Reply:
Works too!